inc/itilsolution.class.php

```diff
@@ -306,19 +307,18 @@ static function messageWarningSolution($params) {
     *
     * @return array
    **/
-   static function deleteAddSolutionButtton($params) {
+   static function deleteAddSolutionButton($params) {
 
       if (isset($params['item'])) {
          $item = $params['item'];
          if ($item->getType() == 'ITILSolution') {
             $warnings = self::checkWarnings($params);
             if (count($warnings)) {
-               $params['options']['canedit'] = false;
-               return $params;
+               echo Html::scriptBlock("$(document).ready(function(){
+                        $('.itilsolution').children().find(':submit').hide();
+                     });");
             }
          }
       }
    }
-
-
```

Comment:
it's your PR (https://github.com/yllen/behaviors/commit/bbf00af52e55d4e03b074061097f512b08b921d0)
so OK, done